sources/packages/api/src-sql/EducationPrograms/Create-education-programs.sql

```diff
@@ -0,0 +1,64 @@
+CREATE TABLE IF NOT EXISTS education_programs(
+  id SERIAL PRIMARY KEY,
+  program_name VARCHAR(300) NOT NULL,
+  program_description VARCHAR(300) NOT NULL,
```

Comment:
will 300 be enough for the description?

Reply:
I checked with @abschwenker and @alpivett and we are going to increase to 500.

Reply:
Updated to 500.